### PR TITLE
lpParser: better handling of quantifiers

### PR DESCRIPTION
**Pull request by fblanqui**
proposal for fixing #430

### CODE REVIEW

**Comment by fblanqui** [comment]
@gabrielhdt @yannl35133 @QGarchery Thanks for your comments on this PR.
make doesn't print any conflict in the grammar. How to make sure that this is indeed the case?

**Comment by fblanqui** [comment]
The file _build/default/src/parsing/lpParser.conflicts is empty.

**Comment by QGarchery** [comment]
I don't know precisely how lambdapi is built but a call to `menhir --explain lpParser.conflict` in `src/parsing` does indeed produce an empty file `lpParser.conflicts` which indicates that there is no conflict.

Thank you for this PR !

File: src/parsing/lpParser.mly [hunk 1]
```diff
@@ -325,13 +306,11 @@ aterm:
   | L_CU_BRACKET t=term R_CU_BRACKET { make_pos $sloc (P_Expl(t)) }
   | n=INT { make_pos $sloc (P_NLit(n)) }
 
-sterm:
-  | t=sterm u=aterm { make_pos $sloc (P_Appl(t,u)) }
+saterm:
```
Comment (gabrielhdt) [inline src/parsing/lpParser.mly:309]: Why `saterm`? I mean `sterm` and `aterm` refer to "atomic" terms and "symbolic" terms, is there a meaning being "saterm"?

Comment (fblanqui) [reply src/parsing/lpParser.mly:309]: I thought sterm = sequence of term's. So saterm = sequence of aterm's.

**Pull request merged**